include/gridtools/sid/delegate.hpp

```diff
@@ -25,36 +26,39 @@ namespace gridtools {
          * @tparam Sid a object that models `SID` concept.
```

Comment:
Can you emphasize somewhere in the comments, e.g. here, that users have to be careful with value vs reference semantics in the delegates.

Reply:
done